ruby_on_rails/create_application_server_deploio.md

```diff
@@ -0,0 +1,35 @@
+# Setup Application Server for Deploio
+
+## Prerequisites
+
+Before setting up your application, ensure you have completed the following for Deploio.
+
+### Prerequisites for Deploio
+
+- You've [read about what Deploio is](https://docs.nine.ch/docs/deplo-io/getting-started-with-deploio).
```

Comment:
This will point to the user guide later. @oliveranthony17 can you please add yourself a go live task for the user guide?